doc/ld-mapping.md

```diff
@@ -1,49 +1,49 @@
-* Query it using SPARQL
+* As data model we used RDF.
+* To describe the tabular data in a re-useable way we used the Data Cube Vocabulary.
+* The data was loaded to a so called triplestore database (provided by LINDAS).
+* From there, we queried the data using SPARQL.
 
 [RDF](http://www.w3.org/TR/rdf11-concepts/), [RDF Data Cube Vocabulary](https://www.w3.org/TR/vocab-data-cube/) and [SPARQL](https://www.w3.org/TR/sparql11-query/) are W3C standards and are implemented in a large variety of open source and commercial products.
 
 We will not cover the basics of RDF within this document, please refer to the [RDF Primer](https://www.w3.org/TR/rdf11-primer/) for a basic introduction to RDF and/or the [Linked Data on Speed](http://presentations.zazuko.com/LD-Speed/#2) presentation. We also provide a list of [recommended books](http://presentations.zazuko.com/LD-Speed/#46) about the topic and implementations of the RDF model in [various programming languages](http://presentations.zazuko.com/LD-Speed/#47).
 
 ![RDF Data Model](img/AThing2.png)
 
-One of the basic concepts of RDF is that a information is identified as a URI, as shown in the RDF Primer. By making this URI available as a "link" on the Web, people can follow the relationships in the data and find related information easily. RDF is thus also often called Linked Data and extends the classical Web of document in a powerful way.
+One of the basic concepts of RDF is that any piece of information is expressed with the help of URIs, as shown in the RDF Primer. By making these URIs available as "links" on the Web, people can follow the relationships in the data and find related information easily. RDF is thus also often called Linked Data and extends the classical Web of documents in a powerful way.
 
 RDF provides support for multiple languages within its core. This is another feature we will use as we can describe meta data in the data cube in multiple languages with almost no extra effort.
 
 ### RDF Data Cube Vocabulary
 
-Within the Linked Data community [Data Cube Vocabulary](http://www.w3.org/TR/vocab-data-cube/) is the most used and most useful vocabulary for converting multi dimensional tabular data to Linked Data and RDF. We just provide a basic introduction to this vocabulary, it is recommended to read the full specification and its example first. Within this prototype we do not provide a fully validated implementation of the Data Cube vocabulary but focus on the most important aspects of it.
+Within the Linked Data community [Data Cube Vocabulary](http://www.w3.org/TR/vocab-data-cube/) is the most used and most useful vocabulary for converting multi dimensional tabular data to Linked Data and RDF. We just provide a basic introduction to this vocabulary; it is recommended to read the full specification and its example first. Within our prototype we did not provide a fully validated implementation of the Data Cube vocabulary, but focused on the most important aspects of it.
```

Comment:
What else would be needed for a fully validated implementation of the Data Cube vocabulary? To what extent would this be relevant?